packages/airnode-node/src/config/types.ts

```diff
@@ -113,5 +121,6 @@ export interface Config {
   readonly nodeSettings: NodeSettings;
   readonly ois: OIS[];
   readonly triggers: Triggers;
+  readonly templates?: Templates;
```

Comment:
I'd prefer this to be an array of templates and the `templateId` to be inside the object. This is more consistent with what we have in `config.json` now and also a bit more explicit, since it is not clear what the record keys should be (from the first look).

Reply:
> it is not clear what the record keys should be (from the first look).

For this reason I'm starting to dislike using `Record<keyType, valueType>` type in favor of the old `[usefulKeyName: keyType]: valueType` syntax (can't remember the name 😁 )

Reply:
Yeah, but that is still only a dev convenience. If you imagine an user who just looks at the config.json, they see a random keys and have to understand Airnode in depth to know that it's the templateId.

Reply:
Yes, I definitely see your point but I'm guessing Vekil did it this way to be consistent with [Airkeeper](https://github.com/api3dao/airkeeper/blob/main/config/airkeeper.example.json#L46) and [Airseeker](https://github.com/api3dao/airseeker/blob/main/config/airseeker.example.json#L66) config files right?

Reply:
Yep, but it's a simple conversion to an array if we want (and also now that we can do the whole verification in [validator](https://github.com/api3dao/airnode/pull/1101#discussion_r875852788)) :+1: